File: bronze/11050/python/main.py

def fectorial(n):
    return n * fectorial(n -1) if (n > 1) else 1

if (__name__ == "__main__"):
    n, k = map(int, input().split())
    print(int(fectorial(n) / (fectorial(k) * fectorial(n - k))))
    

File: bronze/11050/c++/main.cpp
#include <iostream>

int fectorial(int n)
{
    if (n <= 1) {
        return 1;
    }
    return n * fectorial(n - 1);
}

int main()
{
    int n, k;
    std::cin >> n >> k;
    
    std::cout << fectorial(n) / (fectorial(k) * fectorial(n - k));


    return 0;
}